nbs/tutorial_colab.ipynb

```diff
@@ -0,0 +1,405 @@
+{
```

Comment:
<p>This formatting did not come out right on the rendered docs.  Maybe you wanted to indent the text after each number?  This could be formatted better.  </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/398/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>